documentation/modules/overview/con-key-features-kafka.adoc

```diff
@@ -0,0 +1,24 @@
+// This module is included in:
+//
+// overview/assembly-key-features.adoc
+
+[id="key-features-kafka_{context}"]
+= Kafka capabilities
+
+The underlying data stream-processing capabilities and component architecture of Kafka can deliver:
+
+* Allows microservices and other applications to share data with extremely high throughput and low latency
```

Comment:
This reads a bit weird ... when you read it is _... Kafka can deliver Allows microservices and other applications ..._ ... maybe this would sound better?

```suggestion
* Microservices and other applications integration to share data with extremely high throughput and low latency
```